avocado/utils/service.py

```diff
@@ -118,8 +118,9 @@ def method(cmd_result):
             # If service is stopped, exit_status is also not zero.
             # So, we can't use exit_status to check result.
             output = cmd_result.stdout.lower()
```

Comment:
@ldoktor, I moved that line into `re.search`. Could plz help check if this is what you meant? Thanks.